Matthew

PhoenixBlade15 opened the pull request:
Added checking of gen eds.

PhoenixBlade15 committed:
Fixed requesting

PhoenixBlade15 committed:
Added genEd1 hours functionality

Returns the correct hours for gen ed 1.

PhoenixBlade15 committed:
Added Gen Ed Requirement checks

Added the methods to check that all gen eds have been accomplished.

PhoenixBlade15 committed:
Fixed incorrect value

Total credit hours was 0 instead of 6 for gen ed 2, fixed now.

PhoenixBlade15 committed:
Removed h2 from branch temporarily

PhoenixBlade15 committed:
Readding of h2 files

yossarianirving commented:
Before adding this to master, I think the genEd stuff should be in it's  own controller class. Possibly "RequirementsController.java". Also I think a single request for all gen eds is prefered over requests for individual ones. Additionally, database files should not be in the commit since it is impossible to merge them

PhoenixBlade15 committed:
Added more requirement checking functions

Added functions for checking Writing Intensive, ACCT, Upper Division, and Core courses if they are finished or not. Moved all these and gen ed checking function to RequirementsController.java

PhoenixBlade15 committed:
Function addition - Checking General Supporting Courses

Added a function that will have all the concentration supporting courses. The General concentration portion of this is added.

PhoenixBlade15 committed:
Added Digital Forensics and Cybersecurity to function

Digital Forensics and Cybersecurity has been added to the function to check for supporting courses.

PhoenixBlade15 committed:
FInished Supporting Courses Checking function

Added remaining concentrations to the function to make sure all are accounted for.

PhoenixBlade15 committed:
Fixing getCoreRemaining

Noticed I left out core courses and concentrations. Going back and fixing those.
